logger/src/lib.rs

```diff
@@ -95,6 +56,46 @@ impl<D> Logger for Service<D>
 where
     D: Dal + Send + Sync + 'static,
 {
+    async fn store_logs(
+        &self,
+        request: Request<StoreLogsRequest>,
+    ) -> Result<Response<StoreLogsResponse>, Status> {
+        request.verify(Scope::DeploymentPush)?;
```

Comment:
I don't think we have to check the claim - nor the runtime or our own services will have one.

Reply:
Removed the claim.